projects/components/src/select/select.component.ts

```diff
@@ -324,6 +324,10 @@ export class SelectComponent<V> implements ControlValueAccessor, AfterContentIni
     this.propagateControlValueChangeOnTouch = onTouch;
   }
 
+  public setDisabledState(isDisabled?: boolean): void {
```

Comment:
Since the property is public is there a need for a method to access it?

Reply:
I have no clear the reason. Probably @anandtiwary can clarify it. I followed the same structure already implemented in the input component to the Control Value Accessor. I will try to find the reason.

Reply:
Actually this is related to `ControlValueAccessor` interface, this function will be used by reactive form field to set the disabled state using form group itself. We will not pass any [disabled] input in the template while using the select component in the forms.

Reply:
So @itssharmasandeep , does it fine to keep it public?

Reply:
Yeah it is fine.